google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/SpannerDialect.java

```diff
@@ -338,28 +337,14 @@ public String getAddColumnString() {
     return "ADD COLUMN";
   }
 
-  @Override
-  public String getDropForeignKeyString() {
-    throw new UnsupportedOperationException("Cannot drop foreign-key constraint because "
-        + "Cloud Spanner does not support foreign keys.");
-  }
-
   @Override
   public String getAddForeignKeyConstraintString(String constraintName,
       String[] foreignKey,
       String referencedTable,
       String[] primaryKey,
       boolean referencesPrimaryKey) {
-    throw new UnsupportedOperationException("Cannot add foreign-key constraint because "
-        + "Cloud Spanner does not support foreign keys.");
-  }
-
-  @Override
-  public String getAddForeignKeyConstraintString(
-      String constraintName,
-      String foreignKeyDefinition) {
-    throw new UnsupportedOperationException("Cannot add foreign-key constraint because "
-        + "Cloud Spanner does not support foreign keys.");
+    return super.getAddForeignKeyConstraintString(
+        constraintName, foreignKey, referencedTable, primaryKey, false);
```

Comment:
You can remove this method override altogether. It's not doing anything anymore.

Reply:
Can't, I'm overriding the last boolean value to `false`.